test-cases/performance/perf-regression-latency-250gb-with-nemesis.yaml

```diff
@@ -1,12 +1,12 @@
 test_duration: 680
-prepare_write_cmd: ["cassandra-stress write no-warmup cl=ALL n=62500000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=200 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=1..62500000",
-                    "cassandra-stress write no-warmup cl=ALL n=62500000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=200 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=62500001..125000000",
-                    "cassandra-stress write no-warmup cl=ALL n=62500000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=200 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=125000001..187500000",
-                    "cassandra-stress write no-warmup cl=ALL n=62500000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=200 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=187500001..250000000"]
+prepare_write_cmd: ["cassandra-stress write no-warmup cl=ALL n=62500 -schema 'replication(factor=3)' -mode cql3 native -rate threads=200 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=1..62500",
```

Comment:
Is this a TEST only change so you can speed up the test during development?

Reply:
Yes